homeassistant/components/statistics/sensor.py

```diff
@@ -14,6 +14,7 @@
 from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR_DOMAIN
 from homeassistant.components.recorder import get_instance, history
 from homeassistant.components.sensor import (
+    DEVICE_CLASS_STATE_CLASSES,
```

Comment:
The pylint plugin is wrong here, let's silence it for now

```suggestion
    DEVICE_CLASS_STATE_CLASSES,  # pylint disable-next=hass-deprecated-import
```

Reply:
Opened #88172

Reply:
I suggest we revert this PR until this has been addressed.

Reply:
Agreed. Looks like this is resolved now.